src/grb_attrs.jl

```diff
@@ -96,6 +96,24 @@ function get_intattrarray!(r::Array{Cint}, model::Model, name::String, start::In
     r
 end
 
+function get_intattrlist!(r::Array{Cint}, model::Model, name::String, inds::Vector{I}) where {I<:Integer}
```

Comment:
Either drop support for 0.5 or avoid the `where` syntax

Reply:
Probably better to avoid the `where` syntax and make a separate PR later to update this everywhere in the package

Reply:
[FemToCleaner](https://github.com/JuliaComputing/FemtoCleaner.jl) can take care of that